src/doc_config/mod.rs

```diff
@@ -0,0 +1,16 @@
+//! Handles the configuration of how documentation comments should be
+//! generated by `doc-gen`.
+//!
+//! Documentation will likely be written in a TOML syntax and deserialized/validated
```

Comment:
should this say documentation config or template rather than just "documentation"?

Reply:
Definitely should say config files. Thanks!